template.go

```diff
@@ -19,25 +19,26 @@ var (
 	templateDir embed.FS
 	// FuncMap contains extra template functions used by ogent.
 	FuncMap = template.FuncMap{
-		"eagerLoad":      eagerLoad,
-		"edgeOperations": entoas.EdgeOperations,
-		"edgeViewName":   entoas.EdgeViewName,
-		"convertTo":      convertTo,
-		"hasParams":      hasParams,
-		"hasRequestBody": hasRequestBody,
-		"httpVerb":       httpVerb,
-		"httpRoute":      httpRoute,
-		"isCreate":       isCreate,
-		"isRead":         isRead,
-		"isUpdate":       isUpdate,
-		"isDelete":       isDelete,
-		"isList":         isList,
-		"kebab":          strcase.KebabCase,
-		"nodeOperations": entoas.NodeOperations,
-		"replaceAll":     strings.ReplaceAll,
-		"setFieldExpr":   setFieldExpr,
-		"viewName":       entoas.ViewName,
-		"viewNameEdge":   entoas.ViewNameEdge,
+		"eagerLoad":       eagerLoad,
+		"edgeOperations":  entoas.EdgeOperations,
+		"edgeViewName":    entoas.EdgeViewName,
+		"convertTo":       convertTo,
+		"hasParams":       hasParams,
+		"hasRequestBody":  hasRequestBody,
+		"httpVerb":        httpVerb,
+		"httpRoute":       httpRoute,
+		"isCreate":        isCreate,
+		"isRead":          isRead,
+		"isUpdate":        isUpdate,
+		"isDelete":        isDelete,
+		"isList":          isList,
+		"kebab":           strcase.KebabCase,
+		"nodeOperations":  entoas.NodeOperations,
+		"replaceAll":      strings.ReplaceAll,
+		"setFieldExpr":    setFieldExpr,
+		"viewName":        entoas.ViewName,
+		"viewNameEdge":    entoas.ViewNameEdge,
+		"fieldAnnotation": entoas.FieldAnnotation,
```

Comment:
Please order alphabetically

Reply:
done. a few other were out of order as well